Critical Error Rider 2022.3.1

Jetbrains Rider 2022.3.1 error log:

```
com.intellij.diagnostic.PluginException: me.fornever.avaloniarider.statistics.PreviewerUsageLogger PluginClassLoader(plugin=PluginDescriptor(name=AvaloniaRider, id=avalonia-rider, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\Rider2022.3\plugins\avaloniarider, version=1.0.1.99, package=null, isBundled=false), packagePrefix=null, instanceId=34, state=active)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:1014)
	at com.intellij.internal.statistic.service.fus.collectors.FUCounterUsageLogger.lambda$instantiateCounterCollectors$1(FUCounterUsageLogger.java:86)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processWithPluginDescriptor(ExtensionPointImpl.java:299)
	at com.intellij.openapi.extensions.ExtensionPointName.processWithPluginDescriptor(ExtensionPointName.java:153)
	at com.intellij.internal.statistic.service.fus.collectors.FUCounterUsageLogger.instantiateCounterCollectors(FUCounterUsageLogger.java:84)
	at com.intellij.internal.statistic.service.fus.collectors.FUCounterUsageLogger.logRegisteredGroups(FUCounterUsageLogger.java:102)
	at com.intellij.internal.statistic.service.fus.collectors.FUCounterUsageLogger.lambda$new$0(FUCounterUsageLogger.java:69)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:275)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.ClassNotFoundException: me.fornever.avaloniarider.statistics.PreviewerUsageLogger PluginClassLoader(plugin=PluginDescriptor(name=AvaloniaRider, id=avalonia-rider, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\Rider2022.3\plugins\avaloniarider, version=1.0.1.99, package=null, isBundled=false), packagePrefix=null, instanceId=34, state=active)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doLoadClass(ComponentManagerImpl.kt:1519)
	at com.intellij.serviceContainer.ComponentManagerImplKt.access$doLoadClass(ComponentManagerImpl.kt:1)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:1006)
	... 16 more
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.0.2] - 2022-12-28
 ### Fixed
 - [#254](https://github.com/ForNeVeR/AvaloniaRider/issues/254): Error on startup involving `PreviewerUsageLogger`
 
@@ -25,4 +25,5 @@ The initial plugin release. The plugin provides the following features for JetBr
 
 [1.0.0]: https://github.com/ForNeVeR/AvaloniaRider/releases/tag/v1.0.0
 [1.0.1]: https://github.com/ForNeVeR/AvaloniaRider/compare/v1.0.0...v1.0.1
-[Unreleased]: https://github.com/ForNeVeR/AvaloniaRider/compare/v1.0.1...HEAD
+[1.0.2]: https://github.com/ForNeVeR/AvaloniaRider/compare/v1.0.1...v1.0.2
+[Unreleased]: https://github.com/ForNeVeR/AvaloniaRider/compare/v1.0.2...HEAD

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Fixed
+- [#254](https://github.com/ForNeVeR/AvaloniaRider/issues/254): Error on startup involving `PreviewerUsageLogger`
+
 ## [1.0.1] - 2022-12-07
 ### Changed
 - The plugin is now only compatible with Rider 2022.3.1.

--- a/gradle.properties
+++ b/gradle.properties
@@ -1,6 +1,6 @@
 kotlin.stdlib.default.dependency=false
 
-pluginVersionBase=1.0.1
+pluginVersionBase=1.0.2
 # https://www.jetbrains.com/intellij-repository/snapshots/
 riderSdkVersion=2022.3.1
 buildRelease=false

--- a/gradle.properties
+++ b/gradle.properties
@@ -2,5 +2,5 @@ kotlin.stdlib.default.dependency=false
 
 pluginVersionBase=1.0.1
 # https://www.jetbrains.com/intellij-repository/snapshots/
-riderSdkVersion=2022.3-SNAPSHOT
+riderSdkVersion=2022.3.1
 buildRelease=false

--- a/src/rider/main/resources/META-INF/plugin.xml
+++ b/src/rider/main/resources/META-INF/plugin.xml
@@ -24,7 +24,5 @@
                              instance="me.fornever.avaloniarider.idea.settings.AvaloniaSettingsConfigurable" />
 
         <rider.xaml.preview.editor implementation="me.fornever.avaloniarider.idea.editor.AvaloniaPreviewerXamlEditorExtension"/>
-
-        <statistics.counterUsagesCollector implementationClass="me.fornever.avaloniarider.statistics.PreviewerUsageLogger" />
     </extensions>
 </idea-plugin>

